src/language/parser.ts

```diff
@@ -199,9 +208,12 @@ export function parseType(
  */
 export function parseSchemaCoordinate(
   source: string | Source,
-  options?: ParseOptions,
+  options: ParseOptions & { noIgnoredTokens?: true } = {},
```

Comment:
I didn't spot this before, but `parseSchemaCoordinate` should have its own options as they won't overlap with the main options hardly at all:

- `noIgnoredToken` - will always be true
- `noLocation` - the only relevant option?
- `maxTokens` - the grammar ends so rapidly there's literally no value in this
- `experimentalFragmentArguments` - irrelevant

```suggestion
  options?: { noLocation?: boolean | undefined },
```

Reply:
yeah makes sense